### PR TITLE
feat: Enable CI builds

### PR DESCRIPTION
**Pull request by felixrieseberg**
This PR has us build Old Joe in the cloud

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - ps: cd ../..
+  - ps: Tree ./static/data /F
+
+
+
+cache:
+  - '%APPDATA%\npm-cache -> appveyor.yml'
+
+test_script:
+  - node --version
```
Comment (cvaldez98) [inline .appveyor.yml]: Sooo this is just kinda setting it up, we're not yet actually adding any tests right?

Comment (felixrieseberg) [reply .appveyor.yml]: At least not right now, but we should probably eventually at least run `prettier`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    # Apple Worldwide Developer Relations Certification Authority
+    security import ./assets/certs/apple.cer -k ~/Library/Keychains/$KEYCHAIN -T /usr/bin/codesign
+    # Developer Authentication Certification Authority
+    security import ./assets/certs/dac.cer -k ~/Library/Keychains/$KEYCHAIN -T /usr/bin/codesign
+    # Developer ID Felix
+    security import $CERTIFICATE_P12 -k $KEYCHAIN -P $MACOS_CERT_PASSWORD -T /usr/bin/codesign 2>&1 >/dev/null;
+    rm $CERTIFICATE_P12;
+    security set-key-partition-list -S apple-tool:,apple: -s -k travis $KEYCHAIN
+    # Echo the identity
+    security find-identity -v -p codesigning
```
Comment (cvaldez98) [inline .travis.yml]: Not really sure how to interpret this. Could you explain what these lines do? What is the certificate for?

Comment (felixrieseberg) [reply .travis.yml]: Sure! Here's a quick intro https://electronjs.org/docs/tutorial/code-signing

tl;dr: Think of "code signing" like HTTPS for software. We sign our code to prove to the operating system that "Felix Rieseberg" as verified by DigiCert is the author of the software and that it wasn't modified since it was built.
